docs/examples/pricing-tables/pricing.css

```diff
@@ -0,0 +1,20 @@
+body {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+.pricing .card {
```

Comment:
Rather than using descendant selectors like this, we generally prefer to use dedicated classes instead (e.g. `.pricing-card {`, `.pricing-card-title {`)